types/process.go

```diff
@@ -19,6 +19,7 @@ package types
 
 import "time"
 
+// Process is the interface type for returning information on a process
```

Comment:
Maybe interfaces should be documented more like if they wrap methods, as for example in https://golang.org/pkg/io, something like:
```suggestion
// Process is the interface that wraps methods that return information on a process
```

Reply:
Yah, I was kinda waffling on the language since this library is kinda weird with its use of interfaces.